0100    javascript-loops/main.js

```diff
@@ -0,0 +1,118 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            ,
+            doubleAll,
+            ,
+
+ */
+
+// getNumbersToTen
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+
+    currentNumber++;
+  }
+
+  return numbers;
+}
+
+console.log('Value of getNumbersToTen(): ', getNumbersToTen());
+
+// getEvenNumbersToTwenty
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+
+    currentNumber += 2;
+  }
+
+  return evenNumbers;
+}
+
+console.log('Value of getEvenNumbersToTwenty(): ', getEvenNumbersToTwenty());
+
+// repeatWord
+function repeatWord(
+  word,
+  times
+) {
+  let count = 1;
+  let repeated = '';
+
+  while (count <= times) {
+    repeated = repeated + word + ' ';
+
+    count++;
+  }
+
+  return repeated;
+}
+
+console.log('Value of repeatWord(): ', repeatWord('Javascript', 30));
+
+// logEachCharacter
+function logEachCharacter(string) {
+  let i;
+
+  for (i = 0; i < string.length; i++) {
```

Comment:
this works but in the future just declare `i` in the for loop like this `for (let i = 0; i < string.length; i++)`